app/keygen/cmd/main.go

```diff
@@ -0,0 +1,29 @@
+/*
+|    Protect your secrets, protect your sensitive data.
+:    Explore VMware Secrets Manager docs at https://vsecm.com/
+</
+<>/  keep your secrets… secret
+>/
+<>/' Copyright 2023–present VMware, Inc.
+>/'  SPDX-License-Identifier: BSD-2-Clause
+*/
+
+package main
+
+import (
+	"fmt"
+	"github.com/vmware-tanzu/secrets-manager/core/crypto"
+)
+
+func main() {
+	privateKey, publicKey, aesSeed, err := crypto.GenerateKeys()
+
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Just a minor (to make it consistent), other than this, everything looks good to me.

```
if err != nil {
    fmt.Println("Failed to generate keys:")
    fmt.Println(err.Error())
    return
}
```

Reply:
Noted… will update.